tasks.py

```diff
@@ -22,6 +24,11 @@ def genspider(ctx, name, domain):
     test_filename = _gen_tests(name, domain)
     print('Created {0}'.format(test_filename))
 
+    if start_urls:
+        html_filenames = _gen_html(name, start_urls)
```

Comment:
Ha, right! Damn. You could require the list to be quoted, as well... `"http://espn.com|http://npr.org"`... 

I do like splitting on `,http` -- I think you'd want that rather than `,http://` because then it would include https.